aiida/restapi/resources.py

```diff
@@ -209,9 +209,9 @@ def get(self, id=None, page=None):  # pylint: disable=redefined-builtin,invalid-
 
 class QueryBuilder(BaseResource):
     """
-    Representation of a QueryBuilder REST API resource (instantiated with a queryhelp JSON).
+    Representation of a QueryBuilder REST API resource (instantiated with a serialised QueryBuilder instance).
 
-    It supports POST requests taking in JSON :py:func:`~aiida.orm.querybuilder.QueryBuilder.queryhelp`
+    It supports POST requests taking in JSON :py:func:`~aiida.orm.querybuilder.QueryBuilder.as_dict`
```

Comment:
Maybe this should reference the new `QueryDict` type instead then? While also referencing the `as_dict()` method as the "driver"?

Reply:
probably might as well stretch to closing #4660 here as well

Reply:
and then you can reference that